src/screens/PaymentRequestDetail.js

```diff
@@ -114,7 +114,13 @@ class PaymentRequestDetail extends React.Component {
           }}
           >
             <QRCode
-              value={JSON.stringify({ address: `hathor:${this.props.address}`, amount: this.props.amount, token: this.props.token })}
+              value={JSON.stringify({
+                address: `hathor:${this.props.address}`,
+                // amount is a bigint, so we need to stringify it as
+                // it is not serializable
+                amount: this.props.amount.toString(),
+                token: this.props.token
```

Comment:
Why not use the serializer we build in this PR?

Reply:
Because we don't want it to be prefixed with `__BIGINT__:`